google/cloud/sql/connector/rate_limiter.py

```diff
@@ -19,24 +19,16 @@
 
 
 class AsyncRateLimiter(object):
-    """
-    An asyncio-compatible rate limiter which uses the Token Bucket algorithm
-    (https://en.wikipedia.org/wiki/Token_bucket) to limit the number of function calls over a time interval using an event queue.
-
-    :type max_capacity: int
-    :param: max_capacity:
-        The maximum capacity of tokens the bucket will store at any one time.
-        Default: 1
-
-    :type rate: float
-    :param: rate:
-        The number of tokens that should be added per second.
-
-    :type loop: asyncio.AbstractEventLoop
-    :param: loop:
-        The event loop to use. If not provided, the default event loop will be used.
-
+    """An asyncio-compatible rate limiter which uses the Token Bucket algorithm
+    (https://en.wikipedia.org/wiki/Token_bucket) to limit the number 
+    of function calls over a time interval using an event queue.
 
+    Args:
+        max_capacity (int): The maximum capacity of tokens the bucket will store at any one time.
```

Comment:
nit: line length looks a little long, can we try and keep lines to 80 characters and use a second line if need be.